docs/DefiningLabels.md

```diff
@@ -0,0 +1,111 @@
+# Defining Labels
+
+Unless mentioned otherwise, **labels are applied to issues only** (not PRs). 
+
+## Label groups
+
+While GitHub does not have the concept of label groups, we can simulate label groups using systematic label naming. 
+i.e. labels having the same prefix are considered part of a label group.
+
+There are two types of label groups: 
+ 
+* Exclusive groups : No more than one label from the group can be applied to an issue
+* Non-exclusive groups:  Multiple labels from a group can be applied to an issue
+
+**Common label groups**:
+
+* exclusive : `s.` status, `p.` priority, `c.` category, `d.` difficulty, `e.` effort, 
+* non-exclusive:  `a-` aspect, `f-` feature, `t-` tech, 
+ 
+# Common Labels
+
+**Status (`s.`)**
+
+Status labels can be applied to PRs as well.
+
+* `s.OnHold` : The work on the PR has been put on hold pending some other event.
```

Comment:
I'm thinking of trying to operate without status labels.
1. The new 'review status' lowers the need for status tables
2. Status labels often go out of sync with the actual status because some contributors don't have permissions to add labels and others forget to update the label.

The downside is that we cannot filter issues/PRs by status. 

In the meantime we can keep looking for ways to overcome the point 2 above, possibly by automating application of status labels.